content/03.methods.md

```diff
@@ -1,9 +1,33 @@
-#Materials and Methods
+# Materials and Methods
+## Hetionet
+Hetionet [@doi:10.7554/eLife.26726] is a large heterogenous network that contains pharmacological and biological information.
+This network depicts information in the form of nodes and edges of different types: nodes that represent biological and pharmacological entities and edges which represent relationships each entity may share with itself or another. 
+Currently, Hetionet contains 47,031 nodes with 11 different data types and 2,250,197 edges that represent 24 different relationship types (Figure 1).
+Edges in Hetionet were obtained from open databases, such as the GWAS Catalog [@doi:10.1093/nar/gkw1133], DrugBank [@doi:10.1093/nar/gkw1133].
+For this project, we analyzed performance over a subset of the Hetionet relationship types: disease associates with a gene (DaG), compounds binding to a gene (CbG), gene interacts with gene (GiG) and compound treating a disease (CtD).
 
-##Dataset
-Talk about dataset - Pubtator
-Talk about preprocessing Pubtator
-Talk about hand annotations for each realtion
+![Hetionet_metagraph](images/figures/metagraph/metagraph_highlighted_edges.png)
+Figure 1. A metagraph (schema) of Hetionet where pharmacological, biological and disease entities are represented as nodes and the relationships between them are represented as edges.
```

Comment:
Great figure. I like the transparency / fading to highlight which Hetionet relationships this study focuses on.

Should the caption discuss the fading (or whatever you want to call it).

See [proper syntax](https://github.com/manubot/rootstock/blob/master/USAGE.md#figures) for Figure caption (goes where alt text usually goes).